recipes-graphics/imx-g2d/imx-dpu-g2d_2.1.0.bb

```diff
@@ -22,6 +22,10 @@ do_install () {
     cp -Pr ${S}/g2d/usr/include/* ${D}${includedir}
 }
 
+RDEPENDS:${PN}:append:libc-musl = " gcompat"
+
```

Comment:
Please make sure this is consistent on every recipe (close to the end of the recipe and without newlines).